### PR TITLE
csi and ccm upgrade

### PR DESCRIPTION
**Pull request by minchaow**
**How to categorize this PR?**

/area open-source
/kind enhancement
/platform alicloud

**What this PR does / why we need it**:
This PR is to upgrade the version of Alicloud CCM and CSI-plugin.
**Which issue(s) this PR fixes**:
Fixes #
None
**Special notes for your reviewer**:

**Release note**:

```other user
The alibaba-cloud-controller-manager is upgraded to version v1.9.3.372-gcf3535b-aliyun
The csi-plugin-alicloud is upgraded to version v1.18.8.45-1c5d2cd1-aliyun
```

### CODE REVIEW

File: charts/images.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ images:
 - name: alicloud-controller-manager
   sourceRepository: https://github.com/kubernetes/cloud-provider-alibaba-cloud
   repository: registry.eu-central-1.aliyuncs.com/gardener-de/alibaba-cloud-controller-manager
-  tag: v1.9.3-316 # the upstream image is using non-semver tag (registry.cn-shanghai.aliyuncs.com/acs/cloud-controller-manager-amd64:v1.9.3.316-g8daf1a9-aliyun).
+  tag: v1.9.3.372 # the upstream image is using non-semver tag (registry.cn-shanghai.aliyuncs.com/acs/cloud-controller-manager-amd64:v1.9.3.372-gcf3535b-aliyun).
```
Comment (vpnachev) [inline charts/images.yaml:17]:
```suggestion
  tag: v1.9.3-372 # the upstream image is using non-semver tag (registry.cn-shanghai.aliyuncs.com/acs/cloud-controller-manager-amd64:v1.9.3.372-gcf3535b-aliyun).
```

**Review by vpnachev — DISMISSED** [review]

**Review by vpnachev — COMMENTED** [review]
/lgtm code wise

File: charts/images.yaml [hunk 2]
```diff
@@ -48,7 +48,7 @@ images:
 - name: csi-plugin-alicloud
   sourceRepository: https://github.com/AliyunContainerService/csi-plugin
```
Comment (jia-jerry) [inline charts/images.yaml:49]:
```suggestion
  sourceRepository: https://github.com/kubernetes-sigs/alibaba-cloud-csi-driver
```

**Review by jia-jerry — CHANGES_REQUESTED** [review]

**Review by jia-jerry — APPROVED** [review]

**Pull request merged**